sdcm/nemesis.py

```diff
@@ -2075,6 +2077,20 @@ def _prepare_test_table(self, ks='keyspace1', table=None):
             cs_thread = self.tester.run_stress_thread(
                 stress_cmd=stress_cmd, keyspace_name=ks, stop_test_on_failure=False, round_robin=True)
             cs_thread.verify_results()
+            self.stop_nemesis_on_stress_errors(cs_thread)
+
+    def stop_nemesis_on_stress_errors(self, stress_thread: DockerBasedStressThread) -> None:
+        stress_results = super(stress_thread.__class__, stress_thread).get_results()
```

Comment:
Why is the super call needed here ? What are we trying to skip ? And why ?

Reply:
This is to directly call the `get_results` method to get all results from threads. Some command specific `get_results` implementation would filter out failed commands. E.g. in c-s the `get_results` performs:
```
    def get_results(self) -> list[dict | None]:
        ret = []
        results = super().get_results()

        for _, result, event in results:
            if not result:
                # Silently skip if stress command threw error, since it was already reported in _run_stress
                continue
...
        return ret
```

Reply:
It's a bit specific c-s, we should at least put a comment explaining it, it's quite unusual to skip up the MRO like that, I barely understand the reason it's c-s is doing that, let's at least point to the reason we jump it.